src/data/light.json

```diff
@@ -203,17 +203,17 @@
       },
       "border": {
         "base": {
-          "value": "{colors.grey.700}",
+          "value": "{colors.grey.400}",
           "type": "color",
           "description": "Used for the border colour for primary components such as buttons and Message"
```

Comment:
Should we update the desc as well? Since it won't be true for Buttons anyways

Reply:
good catch, should i just keep it generic? "Used for the border colour for primary components"?

Reply:
Looks like it's only being used by CheckableTile for now!